src/main/com/fulcrologic/rad/resolvers_pathom3.cljc

```diff
@@ -0,0 +1,45 @@
+(ns com.fulcrologic.rad.resolvers-pathom3
+  (:require
+    [com.fulcrologic.rad.attributes :as attr]
+    [com.fulcrologic.rad.resolvers-common :as resolvers-common]
+    ; Pathom3 is not a dependency of fulcro-rad, and should not be
```

Comment:
Not a bad idea to put a ns docstring on this ns, and perhaps explain general usage along with this comment.

Reply:
I have written a ns docstring, thank you for the suggestion.